xml/snmp_sysdescr.xml

```diff
@@ -5510,8 +5509,9 @@ Copyright (c) 1995-2005 by Cisco Systems
     <param pos="0" name="os.vendor" value="SonicWall"/>
     <param pos="0" name="os.device" value="Firewall"/>
     <param pos="0" name="os.product" value="SonicOS"/>
-    <param pos="1" name="hw.product"/>
-    <param pos="2" name="os.product"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:sonicwall:sonicos:-"/>
+    <param pos="1" name="hw.family"/>
+    <param pos="2" name="hw.product"/>
```

Comment:
Good typo catch, maybe we should look for double asserts of the same key in the spec?

Reply:
Agreed. I caught it because it broke the CPE script.